index.js

```diff
@@ -5,38 +5,48 @@ const request = require('request');
 const rp = require('request-promise-native');
 const wallpaper = require('wallpaper');
 
-const change = () => {
-    const url = `https://www.reddit.com/r/${randomElement(config.subreddits)}/${config.sort}.json?t=${config.from}limit=${config.limit}`;
-    const options = {
-        uri: url,
-        json: true,
-        simple: true,
-    };
+const maxRetry = config.maxRetry
 
-    const path = config.directory + '/wallpaper';
-    let name = '';
-
-    if (!fs.existsSync(config.directory)) {
-        fs.mkdirSync(config.directory);
+const change = (start) => {
+    if(start === maxRetry) {
+        console.log('We try our best but sorry')
```

Comment:
Let's change the string to: `'We tried our best, but were unable to change the wallpaper. Sorry.'`